src/components/dashboard/main-content/course-enrollments/course-cards/mark-complete-modal/ModalBody.jsx

```diff
@@ -13,11 +13,18 @@ const ModalBody = () => {
     <>
       {confirmError && <ModalError />}
       <p className="m-0">
-        Are you sure you want to mark
+        Are you sure you want to archive
         {' '}
-        <a href={courseLink}>{courseTitle}</a>
+        <a href={courseLink}>{courseTitle}</a>?
         {' '}
-        as complete?
+        You will remain enrolled, but the course will
+        no longer appear as &quot;In Progress&quot;.
+      </p>
+      <p className="mt-1">
+        As long as
+        your license is valid, you can resume the course
```

Comment:
nit: Could this be condensed into a fewer number of lines (within the code, not changing the actual message).